WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/AutofocusTextEditor.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+
+/// Renders a TextEditor with autofocus in iOS 15 and later devices
+///
+struct AutofocusTextEditor: View {
+
+    @Binding var text: String
+
+    @available(iOS 15.0, *)
+    @FocusState private var textEditorIsFocused: Bool
+
+    var body: some View {
+        if #available(iOS 15.0, *) {
+            TextEditor(text: $text)
+                .focused($textEditorIsFocused)
+                .onAppear {
+                    // Without delay '.focused' will not work. This might fix in later releases.
+                    DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
+                        textEditorIsFocused = true
+                    }
```

Comment:
I think you may need to extend this delay a little bit more. The first time I open the view the keyboard is not present, increasing the delay seems to work.

Reply:
Yeah, I have increased the delay to 0.6 and it is working.

Reply:
Sure, I will change it to view modifier, it is a great idea.